### ISSUE TITLE
How do we position the DatePicker on top of the input rather than bottom?

### ISSUE THREAD

**Issue by yadferhad** [elreco/vue-tailwind-datepicker#88]
Currently using this as part of a library, but there's no option to adjust the placement position of the overlay.

**Comment by fabiobulgarella** [comment]
Yes, it would be useful to manually specify the position of the DatePicker (bottom-left, bottom-right, top-left, top-right).

**Comment by haempebe** [comment]
is this done, very grateful if I can get a way to make this happen, I just want my datepicker to go up when the input is at the bottom of the page or vh
